crates/ty_python_semantic/resources/mdtest/typed_dict.md

```diff
@@ -25,12 +25,21 @@ alice: Person = {"name": "Alice", "age": 30}
 reveal_type(alice["name"])  # revealed: Unknown
 # TODO: this should be `int | None`
 reveal_type(alice["age"])  # revealed: Unknown
+
+# TODO: this should reveal `Never`, and it should emit an error
+reveal_type(alice["non_existing"])  # revealed: Unknown
 ```
 
 Inhabitants can also be created through a constructor call:
 
 ```py
 bob = Person(name="Bob", age=25)
+
+reveal_type(bob["name"])  # revealed: str
+reveal_type(bob["age"])  # revealed: int | None
+
+# error: [invalid-key] "Invalid key access on TypedDict `Person`: Unknown key "non_existing""
```

Comment:
I'm not 100% satisfied with the "concise" error message here, but they are constructed from the main message and the message on the primary annotation, so the design space is somewhat limited.